### ISSUE TITLE
No creating directory during import

### ISSUE THREAD

**Issue by Zakoom** [spdermn02/TouchPortal_Discord_Plugin#103]
When import the plugin in TouchPortal it creates the 4 files but instead of creating a directory und the files into it, it creates the files with the directory name and it can't finish the import.

So I need to import it once, rename the files/create the directory und import it again

Only tested it on Mac
<img width="317" alt="Screenshot 2024-11-27 at 19 33 38" src="https://github.com/user-attachments/assets/705d099c-7dfc-47db-8b13-95cbbf1df3a9">

**Comment by spdermn02** [comment]
Thank you for the reminder. We will get this fixed correctly.

**Comment by spdermn02** [comment]
@Zakoom have you imported other plugins on Mac? do those work properly?

**Comment by Zakoom** [comment]
Yes. I just imported a new one to test with the current version, and it worked properly.

**Comment by Zakoom** [comment]
Miss clicked, didn't mean to close

**Comment by BothimTV** [comment]
This happens on Linux (Debian) too

**Comment by spdermn02** [comment]
I'm probably going to be working on moving this to build on specific systems (using github actions) so that it retains correct directory format.  Will update this ticket once that process is complete.

**Comment by spdermn02** [comment]
you mind trying out 5.2.0-Alpha: https://github.com/spdermn02/TouchPortal_Discord_Plugin/releases/tag/v5.2.0-Alpha

**Comment by spdermn02** [comment]
nevermind that isn't going to start properly - working on solution now.

**Comment by spdermn02** [comment]
@BothimTV can you try with a new download from https://github.com/spdermn02/TouchPortal_Discord_Plugin/releases/tag/v5.2.0-Alpha fixed the issues i was seeing
